test/system/auto_complete_component_test.rb

```diff
@@ -7,21 +7,21 @@ def test_renders_component
     with_preview(:default)
 
     assert_selector("auto-complete[for=\"test-id\"][src=\"/auto_complete\"]") do
-      assert_selector("input.form-control")
-      assert_selector("ul[id=\"test-id\"].autocomplete-results", visible: false)
+      assert_selector("input.FormControl-input")
```

Comment:
@jonrohan do you know how I can handle this test with both the Alpha and Beta versions co-existing? I can't figure out how this test runs or which component its looking for 🤔